lua/material/lualine.lua

```diff
@@ -0,0 +1,38 @@
+local colors = vim.g.material_colorscheme_map
+local style = vim.g.material_theme_style
+local notify = vim.notify
+
+-- If material colors are not found, defaults to lualine material theme
+if not colors then
+  local msg = 'material colors not loaded. Defaults to lualine material theme'
+  notify(msg, 'warn', { title = 'material.nvim' })
```

Comment:
It might be confusing to have this be `material.nvim` instead of `material.vim`. Can we change that to match?

Reply:
Sure you're right